### PR TITLE
style(frontend): fixes hero usd balance

### PR DESCRIPTION
**Pull request by BonomoAlessandro**
# Motivation

The `Hero` should always display the text `$ value is not available` if the usd balance can't be load.

# Changes

- usd balance text

# Tests

**before:**
image

**after:**
image

### CODE REVIEW

File: src/frontend/src/lib/components/tokens/TokenExchangeBalance.svelte [hunk 1]
```diff
@@ -14,6 +14,8 @@
 		{formatUSD({ value: usdBalance })}
 	{:else if isNullish(balance)}
 		<span class="animate-pulse">{nullishBalanceMessage ?? '-'}</span>
+	{:else if isNullish(usdBalance) && nonNullish(nullishBalanceMessage)}
+		<span class="animate-pulse">{nullishBalanceMessage}</span>
```
Comment (peterpeterparker) [inline src/frontend/src/lib/components/tokens/TokenExchangeBalance.svelte:18]: Is it correct that it should be animated with a pulse effect?

Comment (AntonioVentilii) [reply src/frontend/src/lib/components/tokens/TokenExchangeBalance.svelte:18]: adding another comment here, I see that above you do `{nullishBalanceMessage ?? '-'}` and you don't use `nonNullish(nullishBalanceMessage)`

while here the contrary

but in the end, those are the same situations no? why not merging the check?

```typescript
{:else if isNullish(balance) || isNullish(usdBalance)}
		<span class="animate-pulse">{nullishBalanceMessage ?? '-'}</span>
```

Comment (BonomoAlessandro) [reply src/frontend/src/lib/components/tokens/TokenExchangeBalance.svelte]: @peterpeterparker yes, it should be animated with a pulse effect. We already use this pulse effect in the check above and here we want to display the same value.

@AntonioVentilii you are right, we can combine those two checks. I just did it and pushed it.

**Review by AntonioVentilii — COMMENTED** [review]
I defer to @peterpeterparker to continue the review, since he was first. All good for the changes I requested on my side

**Review by peterpeterparker — APPROVED** [review]
LGTM, thx

**Review by peterpeterparker — COMMENTED** [review]
Note that I assume we’re going to test this with some E2E tests soon. Otherwise, it would be nice to provide tests for the new feature and to reduce the test debt.

**Pull request merged**